Description: set entry to an empty string instead of undef
 in order to avoid "Use of uninitialized value $entry" warnings.
 Quoting perldelta for 5.26:
   Code like $x = $x . "a" was incorrectly failing to yield a use of
   uninitialized value warning when $x was a lexical variable with an
   undefined value. That has now been fixed. [perl #127877]
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2017-08-08
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=122749
Bug: https://rt.cpan.org/Ticket/Display.html?id=122749

--- a/lib/DPKG/Parse.pm
+++ b/lib/DPKG/Parse.pm
@@ -111,7 +111,7 @@
         die "Cannot find " . $pkg->filename . ", or it's not a file at all!";
     }
     open(STATUS, $pkg->filename);
-    my $entry;
+    my $entry = '';
     my $line_num = -1;
     my $entry_line = 0;
     STATUSLINE: while (my $line = <STATUS>) {
@@ -120,7 +120,7 @@
             my $dpkg_entry = DPKG::Parse::Entry->new('data' => $entry, debug => $pkg->debug, line_num => $entry_line);
             push(@{$pkg->{'entryarray'}}, $dpkg_entry);
             $pkg->{'entryhash'}->{$dpkg_entry->package} = $dpkg_entry;
-            $entry = undef;
+            $entry = '';
             $entry_line = $line_num + 1;
             next STATUSLINE;
         }
